src/Forms/FormField.php

```diff
@@ -1247,17 +1247,19 @@ public function transform(FormTransformation $transformation)
     }
 
     /**
+     * Returns whether the current field has the given class added
+     *
      * @param string $class
      *
-     * @return int
+     * @return bool
      */
     public function hasClass($class)
     {
         $patten = '/' . strtolower($class) . '/i';
```

Comment:
I'm surprised a method this bad still exists; Not that I'm blaming you in any way @robbieaverill :).

I notice it assumes `magic-bananas` contains the class `banana`. =/ Easier to split on whitespace and just use in_array, wouldn't it?

The `static::class` exception is also super wrong and out of date.